module/VuFind/src/VuFind/Controller/HoldsTrait.php

```diff
@@ -52,6 +52,13 @@ trait HoldsTrait
     public function holdAction()
     {
         $driver = $this->loadRecord();
+        //Holds on EDS API records require a different ID
+        //this id can be obtained from the getRtacIdentifier method
+        $id = $driver->getUniqueID();
+        $originalId = $id;
+        if (method_exists($driver, 'getRtacIdentifier')) {
+            $id = $driver->getRtacIdentifier();
+        }
```

Comment:
I believe you could simplify this to:

```suggestion
        $originalId = $driver->getUniqueID();
        $id = $driver->tryMethod('getRtacIdentifier', default: $originalId);
```

Reply:
much cleaner indeed, I've applied this suggestion